### DISCUSSION TITLE
Zip release for python 3.9

### DISCUSSION THREAD

**Discussion by membra** [aws/aws-sdk-pandas#898] [Q&A]
Hello,

I can't see a release zip for python 3.9 here though I see that support was added earlier this year,
https://github.com/awslabs/aws-data-wrangler/releases

Could zip for 3.9 version be added please?

Thanks!

**Comment by jaidisido** [comment]
https://github.com/awslabs/aws-data-wrangler/issues/906#issuecomment-918375407
